lib/ZumoOta/src/FileManager.h

```diff
@@ -15,7 +15,8 @@
 /******************************************************************************
  * Includes
  *****************************************************************************/
-
+ #include <cstdint>

```

Comment:
I don't see why you need these 2 includes here.

Reply:
It will be removed next time.